ase/scripts/generate_ase_environment.py

```diff
@@ -429,6 +429,7 @@ def gen_afu_platform_ifc(json_file):
         cfg.append("--ifc=" + default_ifc)
 
     if (args.platform == 'discrete'):
+        # Compatibility mode for original "discrete" platform argument.
         cfg.append("discrete_pcie3")
```

Comment:
what is discrete_pcie3?

Reply:
The name of the RC platform interface in the Platform Interface Manager.

See https://github.com/OPAE/opae-sdk/blob/master/platforms/platform_db/discrete_pcie3.json